tests/suite/test_virtual_server_upstream_options.py

```diff
@@ -82,6 +82,9 @@ def test_nginx_config_defaults(self, kube_apis, ingress_controller_prerequisites
 
         assert "client_max_body_size 1m;" in config
 
+        assert "zone " in config
```

Comment:
since we don't have a field for configuring zones in VS/VSR, do we need to check for zones in this test?

Reply:
same as above